roles/container-engine/docker/molecule/default/molecule.yml

```diff
@@ -0,0 +1,38 @@
+---
+driver:
+  name: delegated
+lint:
+  name: yamllint
+  # Disabled until issues are fixed
+  enabled: False
+platforms:
+  - name: kubespray-docker
+    image: quay.io/kubespray/vm-ubuntu-1804:latest
+    groups:
+      - k8s-cluster
+provisioner:
+  name: ansible
+  env:
+    ANSIBLE_ROLES_PATH: ../../../../
+  config_options:
+    defaults:
+      callback_whitelist: profile_tasks
+  lint:
+    name: ansible-lint
+    # Disabled until issues are fixed
```

Comment:
Which issues?

Reply:
```$ molecule lint
--> Validating schema /home/anders/git/kubespray/roles/container-engine/docker/molecule/default/molecule.yml.
Validation completed successfully.
--> Test matrix
    
└── default
    └── lint
    
--> Scenario: 'default'
--> Action: 'lint'
Skipping, lint is disabled.
--> Executing Flake8 on files found in /home/anders/git/kubespray/roles/container-engine/docker/molecule/default/tests/...
Lint completed successfully.
--> Executing Ansible Lint on /home/anders/git/kubespray/roles/container-engine/docker/molecule/default/playbook.yml...
    [404] Doesn't need a relative path in role
    /home/anders/git/kubespray/roles/container-engine/containerd/tasks/crictl.yml:7
    Task/Handler: Install crictl config
    
    [701] No 'galaxy_info' found
    /home/anders/git/kubespray/roles/container-engine/docker/meta/main.yml:2
    {'meta/main.yml': {'dependencies': [{'role': 'container-engine/docker-storage', 'when': 'docker_container_storage_setup and ansible_os_family == "RedHat"', '__line__': 3, '__file__': '/home/anders/git/kubespray/roles/container-engine/docker/meta/main.yml'}], '__line__': 2, '__file__': '/home/anders/git/kubespray/roles/container-engine/docker/meta/main.yml'}}
    
    [204] Lines should be no longer than 160 chars
    /home/anders/git/kubespray/roles/container-engine/docker/tasks/main.yml:49
      when: (not ansible_os_family in ["CoreOS", "Coreos", "Container Linux by CoreOS", "ClearLinux"]) and (ansible_kernel is version(docker_kernel_min_version, "<"))
    
    [204] Lines should be no longer than 160 chars
    /home/anders/git/kubespray/roles/container-engine/docker/tasks/main.yml:74
      when: not (ansible_os_family in ["CoreOS", "Coreos", "Container Linux by CoreOS", "RedHat", "Suse", "ClearLinux"] or is_atomic) and (docker_repo_info.repos|length > 0)
    
    [204] Lines should be no longer than 160 chars
    /home/anders/git/kubespray/roles/container-engine/docker/tasks/main.yml:99
        - not (ansible_os_family in ["CoreOS", "Coreos", "Container Linux by CoreOS", "RedHat", "Suse", "ClearLinux"] or is_atomic) and (dockerproject_repo_info.repos|length > 0)
    
    [306] Shells that use pipes should set the pipefail option
    /home/anders/git/kubespray/roles/container-engine/docker/tasks/set_facts_dns.yml:31
    Task/Handler: check system nameservers
    
    [306] Shells that use pipes should set the pipefail option
    /home/anders/git/kubespray/roles/container-engine/docker/tasks/set_facts_dns.yml:37
    Task/Handler: check system search domains
    
    [306] Shells that use pipes should set the pipefail option
    /home/anders/git/kubespray/roles/container-engine/docker/tasks/systemd.yml:14
    Task/Handler: get systemd version
    
    [701] No 'galaxy_info' found
    /home/anders/git/kubespray/roles/download/meta/main.yml:2
    {'meta/main.yml': {'allow_duplicates': True, '__line__': 2, '__file__': '/home/anders/git/kubespray/roles/download/meta/main.yml'}}
    
    [305] Use shell only when shell functionality is required
    /home/anders/git/kubespray/roles/download/tasks/check_pull_required.yml:7
    Task/Handler: check_pull_required |  Generate a list of information about the images on a node
    
    [204] Lines should be no longer than 160 chars
    /home/anders/git/kubespray/roles/download/tasks/download_container.yml:102
          shell: "{{ docker_bin_dir }}/docker save {{ image_reponame }} | gzip -{{ download_compress }} > {{ image_path_cached if download_localhost else image_path_final }}"
    
    [305] Use shell only when shell functionality is required
    /home/anders/git/kubespray/roles/download/tasks/prep_download.yml:13
    Task/Handler: Register docker images info
    
    [305] Use shell only when shell functionality is required
    /home/anders/git/kubespray/roles/download/tasks/prep_download.yml:59
    Task/Handler: prep_download | On localhost, check if user has access to docker without using sudo
    
    [306] Shells that use pipes should set the pipefail option
    /home/anders/git/kubespray/roles/download/tasks/prep_kubeadm_images.yml:43
    Task/Handler: prep_kubeadm_images | Generate list of required images
    
    [305] Use shell only when shell functionality is required
    /home/anders/git/kubespray/roles/download/tasks/set_docker_image_facts.yml:17
    Task/Handler: Register docker images info
    
    [701] No 'galaxy_info' found
    /home/anders/git/kubespray/roles/kubespray-defaults/meta/main.yml:2
    {'meta/main.yml': {'dependencies': [{'role': 'download', 'skip_downloads': True, 'tags': ['facts'], '__line__': 3, '__file__': '/home/anders/git/kubespray/roles/kubespray-defaults/meta/main.yml'}], '__line__': 2, '__file__': '/home/anders/git/kubespray/roles/kubespray-defaults/meta/main.yml'}}
    
```

Reply:
```--> Executing Yamllint on files found in /home/anders/git/kubespray/roles/container-engine/docker/...
    /home/anders/git/kubespray/roles/container-engine/docker/defaults/main.yml
      30:81     error    line too long (100 > 80 characters)  (line-length)
      33:81     error    line too long (86 > 80 characters)  (line-length)
      42:81     error    line too long (82 > 80 characters)  (line-length)
      48:81     error    line too long (88 > 80 characters)  (line-length)
      70:14     error    no new line character at the end of file  (new-line-at-end-of-file)
    
    /home/anders/git/kubespray/roles/container-engine/docker/vars/debian.yml
      9:81      error    line too long (81 > 80 characters)  (line-length)
      15:81     error    line too long (82 > 80 characters)  (line-length)
      16:81     error    line too long (83 > 80 characters)  (line-length)
      17:81     error    line too long (81 > 80 characters)  (line-length)
      23:14     warning  truthy value should be one of [false, true]  (truthy)
    
    /home/anders/git/kubespray/roles/container-engine/docker/vars/ubuntu-arm64.yml
      8:81      error    line too long (81 > 80 characters)  (line-length)
      10:81     error    line too long (82 > 80 characters)  (line-length)
      11:81     error    line too long (83 > 80 characters)  (line-length)
      12:81     error    line too long (81 > 80 characters)  (line-length)
      18:14     warning  truthy value should be one of [false, true]  (truthy)
    
    /home/anders/git/kubespray/roles/container-engine/docker/vars/ubuntu-amd64.yml
      10:81     error    line too long (81 > 80 characters)  (line-length)
      11:81     error    line too long (81 > 80 characters)  (line-length)
      12:81     error    line too long (81 > 80 characters)  (line-length)
      14:81     error    line too long (82 > 80 characters)  (line-length)
      15:81     error    line too long (83 > 80 characters)  (line-length)
      16:81     error    line too long (81 > 80 characters)  (line-length)
      22:14     warning  truthy value should be one of [false, true]  (truthy)
    
    /home/anders/git/kubespray/roles/container-engine/docker/tasks/set_facts_dns.yml
      24:81     error    line too long (85 > 80 characters)  (line-length)
      29:81     error    line too long (92 > 80 characters)  (line-length)
      32:81     error    line too long (94 > 80 characters)  (line-length)
      33:17     warning  truthy value should be one of [false, true]  (truthy)
      35:15     warning  truthy value should be one of [false, true]  (truthy)
      38:81     error    line too long (84 > 80 characters)  (line-length)
      39:17     warning  truthy value should be one of [false, true]  (truthy)
      41:15     warning  truthy value should be one of [false, true]  (truthy)
      45:81     error    line too long (100 > 80 characters)  (line-length)
      50:81     error    line too long (131 > 80 characters)  (line-length)
      55:81     error    line too long (141 > 80 characters)  (line-length)
    
    /home/anders/git/kubespray/roles/container-engine/docker/tasks/systemd.yml
      27:81     error    line too long (97 > 80 characters)  (line-length)
    
    /home/anders/git/kubespray/roles/container-engine/docker/tasks/docker_plugin.yml
      3:81      error    line too long (84 > 80 characters)  (line-length)
    
    /home/anders/git/kubespray/roles/container-engine/docker/tasks/main.yml
      15:81     error    line too long (107 > 80 characters)  (line-length)
      16:81     error    line too long (113 > 80 characters)  (line-length)
      17:81     error    line too long (89 > 80 characters)  (line-length)
      18:81     error    line too long (113 > 80 characters)  (line-length)
      49:81     error    line too long (162 > 80 characters)  (line-length)
      66:81     error    line too long (129 > 80 characters)  (line-length)
      74:81     error    line too long (169 > 80 characters)  (line-length)
      88:81     error    line too long (127 > 80 characters)  (line-length)
      99:81     error    line too long (174 > 80 characters)  (line-length)
      111:81    error    line too long (83 > 80 characters)  (line-length)
      119:81    error    line too long (107 > 80 characters)  (line-length)
      126:15    warning  truthy value should be one of [false, true]  (truthy)
      138:17    warning  truthy value should be one of [false, true]  (truthy)
      139:81    error    line too long (83 > 80 characters)  (line-length)
      147:81    error    line too long (83 > 80 characters)  (line-length)
      163:81    error    line too long (153 > 80 characters)  (line-length)
      201:81    error    line too long (85 > 80 characters)  (line-length)
      230:81    error    line too long (129 > 80 characters)  (line-length)
      232:81    error    line too long (82 > 80 characters)  (line-length)
      238:81    error    line too long (97 > 80 characters)  (line-length)
      250:14    warning  truthy value should be one of [false, true]  (truthy)
```

Reply:
Probably molecule should refer to the global `.ansible-lint` and `.yamllint` files in the root of this repo.

Reply:
Ah, thank you, @MarkusTeufelberger. I will fix it.